src/Controller/QuizTemplateController.php

```diff
@@ -175,10 +176,19 @@ public function editQuizzes (RouteMatch $routeMatch, Request $request) {
         }
 
         $id = $routeMatch->getRequestAttributes()['id'];
+        /**
+         * @var QuizTemplate $quiz
```

Comment:
return type of `quizService->getQuiz()` should be `?QuizTemplate` not `?EntityInterface` 

if you make that change, this `@var` is no longer needed